pandas/tests/tslibs/test_normalize_date.py

```diff
@@ -7,6 +7,8 @@
 
 from pandas._libs import tslibs
 
+from pandas import Timestamp
```

Comment:
Nitpick: can you import this from tslibs?  I like to pretend that tests.tslibs is independent of the rest of pandas.